packages/sitecore-jss/src/editing/models.ts

```diff
@@ -26,3 +26,13 @@ export enum LayoutKind {
   Final = 'final',
   Shared = 'shared',
 }
+
+/**
+ * Represents the kind of chrome element.
```

Comment:
Let's add some mentioning about "Metadata" mode, since it's used only for this purpose (Not related to "Chromes" mode)